homeassistant/components/prometheus/__init__.py

```diff
@@ -829,16 +820,16 @@ def _handle_alarm_control_panel(self, state: State) -> None:
             )
 
             alarm_states = [
-                STATE_ALARM_ARMED_AWAY,
-                STATE_ALARM_ARMED_CUSTOM_BYPASS,
-                STATE_ALARM_ARMED_HOME,
-                STATE_ALARM_ARMED_NIGHT,
-                STATE_ALARM_ARMED_VACATION,
-                STATE_ALARM_DISARMED,
-                STATE_ALARM_TRIGGERED,
-                STATE_ALARM_PENDING,
-                STATE_ALARM_ARMING,
-                STATE_ALARM_DISARMING,
+                AlarmControlPanelState.ARMED_AWAY,
+                AlarmControlPanelState.ARMED_CUSTOM_BYPASS,
+                AlarmControlPanelState.ARMED_HOME,
+                AlarmControlPanelState.ARMED_NIGHT,
+                AlarmControlPanelState.ARMED_VACATION,
+                AlarmControlPanelState.DISARMED,
+                AlarmControlPanelState.TRIGGERED,
+                AlarmControlPanelState.PENDING,
+                AlarmControlPanelState.ARMING,
+                AlarmControlPanelState.DISARMING,
```

Comment:
Side note: Can we iterate the enum directly instead of creating this intermediate list?

Reply:
I have changed this in this PR